File: 백준/Bronze/10953. A＋B － 6/A＋B － 6.cc
#include <stdio.h>
int main(){
    int x, y;
    int count;
    scanf("%d\n", &count);
    for(int i = 0 ; i != count; i++){
        scanf("%d,", &x);
        scanf("%d\n", &y);
        printf("%d\n", x + y);
        }
    }

File: 백준/Bronze/10953. A＋B － 6/A＋B － 6.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in).useDelimiter(",|\n");
        int count = scanner.nextInt();
        for (int i = 0; i < count; i++) {
            int x = scanner.nextInt();
            int y = scanner.nextInt();
            System.out.println(x + y);
        }
        scanner.close();
    }
}
